py/server/deephaven/updateby_clause.py

```diff
@@ -0,0 +1,269 @@
+    update-by operation. """
+    j_object_type = _JEmaControl
+
+    @property
+    def j_object(self) -> jpy.JType:
+        return self.j_ema_control
+
+    def __init__(self, on_null: BadDataBehavior = BadDataBehavior.SKIP,
+                 on_nan: BadDataBehavior = BadDataBehavior.SKIP,
+                 on_null_time: BadDataBehavior = BadDataBehavior.SKIP,
+                 on_negative_deltatime: BadDataBehavior = BadDataBehavior.THROW,
+                 on_zero_deltatime: BadDataBehavior = BadDataBehavior.SKIP,
+                 big_value_context: MathContext = MathContext.DECIMAL128):
+        """Initializes a EmaControl for evaluating EMAs (exponential moving average) with table update-by operation.
+
+        Args:
+            on_null (BadDataBehavior): the behavior for when null values are encountered, default is SKIP
+            on_nan (BadDataBehavior): the behavior for when Nan values are encountered, default is SKIP
+            on_null_time (BadDataBehavior): the behavior for when null timestamps are encountered, default is SKIP
+            on_negative_deltatime (BadDataBehavior): the behavior for when negative sample-to-sample time differences are
+                encountered, default is THROW
+            on_zero_deltatime (BadDataBehavior): the behavior for when zero sample-to-sample-time differences are
+                encountered, default is SKIP
+            big_value_context (MathContext): the context to use when processing big numeric values (BigDecimal/BigInteger).
+                default is DECIMAL128.
+
+        Raises:
+            DHError
+        """
+        try:
+            j_builder = _JEmaControl.builder()
+            self.j_ema_control = (j_builder.onNullValue(on_null.value)
+                                  .onNanValue(on_nan.value)
+                                  .onNullTime(on_null_time.value)
+                                  .onNegativeDeltaTime(on_negative_deltatime.value)
+                                  .onZeroDeltaTime(on_zero_deltatime.value)
+                                  .bigValueContext(big_value_context.value).build())
+        except Exception as e:
+            raise DHError(e, "failed to build a EmaControl object.") from e
+
+
+class UpdateByClause(JObjectWrapper):
+    """A UpdateByClause represents an operator for the table update-by operation."""
+
+    j_object_type = _JUpdateByClause
+
+    def __init__(self, j_updateby_clause):
+        self.j_updateby_clause = j_updateby_clause
+
+    @property
+    def j_object(self) -> jpy.JType:
+        return self.j_updateby_clause
+
+
+def ema_tick_decay(time_scale_ticks: int, pairs: Union[str, List[str]],
+                   ema_control: EmaControl = None) -> UpdateByClause:
+    """Creates an EMA (exponential moving average) UpdateByClause for the supplied column name pairs, using ticks as
+    the decay unit.
+
+    The formula used is
+        a = e^(-1 / timeScaleTicks)
```

Comment:
```suggestion
        a = e^(-1 / time_scale_ticks)
```
Need to use the pythonicarg names.  Check other cases for this.